CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes to this project will be documented in this file.
 -   #1844 : Add line numbers and code to errors from built-in function calls.
 -   \[INTERNALS\] Added `container_rank` property to `ast.datatypes.PyccelType` objects.
 -   \[DEVELOPER\] Added an improved traceback to the developer-mode errors for errors in function calls.
+-   #1754 : Add Python support for set method `update()`.
```

Comment:
Can you move this up to be with the other set methods. I usually try and put the information that is relevant for users first and information for developers (usually just the internals) last

Reply:
Fixed : 0bca7ba